lists/bo/bo-nit.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": {
+    "en": "Tax Identification Number",
+    "local": "Número de Identificación Tributaria "
+  },
+  "coverage": [
+    "BO"
+  ],
+  "description": {
+    "en": "The NIT or Identify Tax Number is the number assigned by the Tax Administration to a natural person, a legal entity at the time of registration for any economic activity, by virtue of which they are taxable persons of some of the taxes established by Law.\nhttps://impuestos.com.bo/rnd-03-13/"
+  },
+  "structure": [
+    "company/sole_trader"
```

Comment:
Did you mean, instead, to select just 'company' here? That would make more sense. (I imagine that Trusts and Charities might also be taxable in some situations.)